client/components/symbol-art-single-post.jsx

```diff
@@ -41,6 +41,10 @@ export default function SinglePostSymbolArt(props) {
     filePropsLayerCount
   } = props.symbolArt;
 
+  if (!tags) {
+    tags = [];
+  }
+
```

Comment:
I'm not certain what your value of `tags` could be but if it's possible undefined, a default value could work.

```js
  const {
    postId,
    title,
    description,
    previewImagePath,
    tags = [],
    username,
    userId,
    filePropsSound,
    filePropsName,
    filePropsLayerCount
  } = props.symbolArt;
```